docs/software_setup/docker.md

```diff
@@ -47,12 +86,14 @@ sudo systemctl enable docker   # enable docker daemon at boot
 ```
 
 Now you can go to ``https://<jetbot_ip>:8888`` from a web browser and start programming JetBot!
+
 You can do this from any machine on your local network.  The password to log in is ``jetbot``.
 
 ![](https://user-images.githubusercontent.com/25759564/92091965-51ae4f00-ed86-11ea-93d5-09d291ccfa95.png)
 
+The `enable.sh` script causes the docker containers to restart at boot. This means the next time you power on your JetBot, the containers will automatically start, and you should see the IP address displayed on the PiOLED display screen. All you need to do is type this into your web browser and start programming!
+
+???+ tip
+    For more information on configuring and using docker with JetBot, check out the [Docker Tips](reference/docker_tips.md) page.
 
```

Comment:
Now that you've finished setting up you're JetBot, you're ready to run through the [examples](examples/basic_motion.md)!